src/lib/menu/menu-trigger.ts

```diff
@@ -216,23 +218,29 @@ export class MdMenuTrigger implements AfterViewInit, OnDestroy {
 
   /** Closes the menu. */
   closeMenu(): void {
+    this.menu.close.emit();
+  }
+
+  /** Focuses the menu trigger. */
+  focus() {
+    this._element.nativeElement.focus();
+  }
+
+  /**
+   * Closes the menu and does the necessary cleanup.
+   */
+  private _closeMenu() {
```

Comment:
One last nitpick: can you rename this to `_destroyMenu` to avoid confusion with `closeMenu`?

Reply:
Done.